api/src/main/java/org/apache/brooklyn/api/effector/ParameterType.java

```diff
@@ -45,4 +47,6 @@
      * @return The default value for this parameter, if not supplied during an effector call.
      */
     public T getDefaultValue();
+
+    public boolean isSensitive();
```

Comment:
Given your question about naming `shouldSanitize` or `isSensitive`, let's mark this as `@Beta`.